frontend/lib/src/hocs/withMapboxToken/MapboxToken.test.ts

```diff
@@ -96,24 +96,7 @@ describe("MapboxToken", () => {
     expect(MapboxToken.token).toBeUndefined()
   })
 
-  xit("Errors if not localhost and missing token", async () => {
-    window.location = { hostname: "https://streamlit.io" } as Location
-    const sessionInfo = createSessionInfo("")
-
-    await expect(MapboxToken.get(sessionInfo)).rejects.toThrow(
-      "No Mapbox token provided"
-    )
-  })
-
-  xit("Errors if not hello.py and missing token", async () => {
-    const sessionInfo = createSessionInfo("", "streamlit run example.py")
-
-    await expect(MapboxToken.get(sessionInfo)).rejects.toThrow(
-      "No Mapbox token provided"
-    )
-  })
-
-  xit("does not error if running hello.py and missing token", async () => {
```

Comment:
Tests related to functionality we no longer intend to implement. See `MapboxToken.ts` file comment